src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml

```diff
@@ -144,7 +144,7 @@
 
   <BoolProperty Name="CheckForOverflowUnderflow"
                 DisplayName="Check for arithmetic overflow"
-                Description="Specifies whether integer arithmetic that results in a value outside the range of the data type, and that is not in the scope of a checked or unchecked keyword, causes a run-time exception."
+                Description="Throw exceptions when integer arithmetic produces out of range values."
```

Comment:
Should be "out-of-range values" as the whole string "out of range" serves as a single adjective modifying "values".